### PR TITLE
Bug 1810196: Use RPM for UPI dependencies

### PR DESCRIPTION
**Pull request by pierreprinetti**
The use of pip is not supported.

### CODE REVIEW

File: docs/user/openstack/install_upi.md [hunk 1]
```diff
@@ -95,13 +95,10 @@ This repository contains [Ansible playbooks][ansible-upi] to deploy OpenShift on
   * netaddr
   * openstackclient
 
-The file `requirements.txt` included in the playbook directory helps using `pip` for gathering the required dependencies in a Python virtual environment:
+This command installs all required rependencies on a RHEL or Fedora Linux distribution:
```
Comment (mandre) [inline docs/user/openstack/install_upi.md]: s/rependencies/dependencies/

**Comment by pierreprinetti** [comment]
> Do we depend on a particular version of one of these packages?
I am not aware of recent changes that affect what we need here.

(We will need to find a solution for `soft-`anti-affinity, but that's not in 4.4)

File: docs/user/openstack/install_upi.md [hunk 2]
```diff
@@ -95,13 +95,10 @@ This repository contains [Ansible playbooks][ansible-upi] to deploy OpenShift on
...
 
-The file `requirements.txt` included in the playbook directory helps using `pip` for gathering the required dependencies in a Python virtual environment:
+This command installs all required dependencies on a RHEL or Fedora Linux distribution:
 
 ```sh
-python -m venv venv
-source venv/bin/activate
-pip install -U pip
-pip install -r requirements.txt
+yum install python-openstackclient ansible python-openstacksdk python-netaddr
```
Comment (racedo) [inline docs/user/openstack/install_upi.md]: When testing this from a RHEL 8 box customers need to know where to install the packages from. RHEL8 also uses python3-$pkgname so we should document this:

sudo subscription-manager register # if not done already
sudo subscription-manager attach --pool=$YOUR_POOLID  # if not done already
sudo subscription-manager repos --disable=*  # if not done already

Then enable the right RHEL8 repos:
sudo subscription-manager repos --enable=rhel-8-for-x86_64-baseos-rpms --enable=openstack-16-tools-for-rhel-8-x86_64-rpms --enable=ansible-2.8-for-rhel-8-x86_64-rpms --enable=rhel-8-for-x86_64-appstream-rpms

Then install the packages:
sudo yum install python3-openstackclient ansible python3-openstacksdk python3-netaddr

Comment (pierreprinetti) [reply docs/user/openstack/install_upi.md]: I have added what you suggest. Do you mind having a look?

**Comment by pierreprinetti** [comment]
Incorporate comment by @zokahn https://bugzilla.redhat.com/show_bug.cgi?id=1810202#c2

> Maybe also add 
> sudo alternatives --set python /usr/bin/python3
> to be able to just call python. Some of the scripts on the install doc use them

**Pull request merged**